.github/workflows/test_interface_portalicious.yml

```diff
@@ -0,0 +1,39 @@
+# See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: 'Test Interface: Portalicious'
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/test_interface_portalicious.yml'
+      - 'interfaces/Portalicious/**'
+
+env:
+  interfacePath: 'interfaces/Portalicious/'
```

Comment:
I think the last slash isn't necessary..

It will at least result in "double slashes" for the `.node-version`-path below...